.github/workflows/rebase-dependabot.yml

```diff
@@ -0,0 +1,25 @@
+name: rebase-dependabot-pr
+on:
+  # push:
+  #   branches: [ main ]
```

Comment:
Ah, you want to test first and enable in a second step? That's wise -- the new workflow has to be on main first to be able to trigger it on the UI.